### ISSUE TITLE
P03-03

### ISSUE THREAD

**Issue by madanvikram** [zackchase/mxnet-the-straight-dope#132]
I understand the tutorial is in an early state, but it is looking great! My detailed feedback is below. Also currently the model is not working because there is a missing argument (last_batch) for DataLoader.

High-level comments:
1) I think the intro is very helpful, but should we consider bringing this forward to P03-C01?
2) I think the flow would be better if the bias-variance tradeoff discussion preceded the discussion of overfitting and regularization.
3) Do we need the softmax function anymore?

Areas that require clarification:
1) We should clarify the following sentence: “Given a network with n nodes we are sampling uniformly at random from the 2^n networks in which a subset of the nodes are turned off.”
2) Why are we using the Gluon DataLoader function in the ‘From Scratch’ tutorial? (I see you recently added this to P03-C01 also)
3) We should explain why the transform function is necessary? (I see you recently added this to P03-C01 also)
4) It would be helpful to discuss why scaling is necessary in the dropout function

Typos / Minor Changes:
1) There is a missing ‘n’ in “Wester Union”
2) You need to add an ‘a’ in “either positive or negative weight”
3) You do not need the one_hot_label line in the evaluate_accuracy function
4) It would be better to create a variable for the dropout rate in the training loop
5) We need to align the moving loss section with previous tutorials
